src/utils/date-helpers.js

```diff
@@ -59,10 +59,42 @@ export const getLocaleData = (locale) => {
   return localeResult;
 };
 
+/**
+ * Returns the string representation for a provided year, month & day.
+ *
+ * @param year {number} - any year
+ * @param month {number} - can be between 1 and 12
+ * @param day {number} - can be between 1 and 31 depending on the month
+ * @returns {string}: a string representing the date in the format yyyy-mm-dd
+ */
 export const getDateKey = (year, month, day) => `${year}-${month}-${day}`;
 
+/**
+ * Returns the date for a date string representation.
+ *
+ * @param year {number} - any year
+ * @param month {number} - can be between 1 and 12
+ * @param day {number} - can be between 1 and 31 depending on the month
+ * @returns {date} - the parse date
+ */
+export const getDateForDateKey = (dateKey) => {
+  const [year, month, day] = dateKey.split('-');
+  return new Date(parseInt(year, 10), parseInt(month, 10) - 1, parseInt(day, 10));
+};
+
```

Comment:
Hey @nikgraf , why add function for something date constructor already does for you.